src/main/java/com/watch/shop/app/controller/MenuConstants.java

```diff
@@ -0,0 +1,29 @@
+package com.watch.shop.app.controller;
+
+public class MenuConstants {
```

Comment:
Previous variant was better

Constants should hold most of String literals in App.
It shouldn't limit to just Menu, it could be other String constants as well,

so just Constants.java , no need to limit name, it's containers for Constants.

You use Constants in Controller. But it could be also possible to use in Service.
Your controller uses Service and View, but you donot put all these layers into controllers?

Constants just one from many components to use, I doubt it's responsibility of controllers, though it could be acceptable in some degree and cases...

There no easy answer where it's better to be, Controller is also one from variant from the perspective it related to Main menu.

But think about Constants not just as for Main menu helper class, but as container for all Constants, for example Exception message etc, in this case it could be wider use than just controller.

so put or in View or in Util package